docs/howtos/extending/magicgui.md

```diff
@@ -178,6 +178,10 @@ The following napari types may be used as *parameter* type annotations in
   {attr}`napari.types.ImageData` or  {attr}`napari.types.LabelsData`
 - {class}`napari.Viewer`
 
+```{note}
+If you're interested in getting synchronized information from the napari viewer into widgets without `@magicgui`, see the [`create_widget` example](#magicgui-widgets-create_widget) below.
+```
```

Comment:
I don't think this is necessary. I think we automatically update for all magicgui registered napari types; layer, layer subclasses and layer type data (e.g., `napari.types.ImageData`). 
I am not sure if there would be any cases where you would want an input that is not in the above list.

If there is, we could mention that for non registered types you need to manually synchronize, see `QWidget` example for more...

cc @DragaDoncila

Reply:
I have made the note more precise. I added it, because I was aware that one can easily create these synchronized dropdown lists of the available layers with `@magicgui`. But then, they were part of a bigger widget, and I was not able to figure out to create only these widgets which is possible with `create_widget`.